release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -290,6 +290,16 @@ For more information on configuration drift, see xref:../post_installation_confi
 
 You can now enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control groups version 2] (cgroups v2) on specific nodes in your cluster. The {product-title} process for enabling cgroups v2 disables all cgroups version 1 controllers and hierarchies. The {product-title} cgroups version 2 feature is in Developer Preview and is not supported by Red Hat at this time.
 
+[id="ocp-4-10-node-health-check-operator"]
+==== Node Health Check Operator enhancements
```

Comment:
not sure if this would be the right place, but we need to mention somewhere that people, who are upgrading an existing installation of NHC, need to update the channel from `alpha` to `candidate` manually, otherwise the upgrade won't happen.

Reply:
@slintes I moved this out of release notes and added it to the [main docs PR](https://github.com/openshift/openshift-docs/pull/41724/files#diff-88f044abbb2615d4b62f74f1465525477e6571d9e193b910016a4b3892df2499R66-R73) to avoid confusion. Is that OK